tutorials/docker-gcplogs-driver/index.md

```diff
@@ -151,6 +151,70 @@ to use the Google Cloud logging driver when it is started by systemd:
     the [Logs Viewer](https://console.cloud.google.com/logs/viewer) as you did
     before.
 
+## Persisting configuration across reboots
+
+On Container-Optimized OS, files in `/etc/` are writable, but [data does not
+persist across
+reboots](/container-optimized-os/docs/concepts/security#filesystem).
+Instead, [use cloud-init to configure Container-Optimized OS
+instances](/container-optimized-os/docs/how-to/create-configure-instance#using_cloud-init).
+
+To configure cloud-init, [update the instance
+metadata](/compute/docs/storing-retrieving-metadata#updatinginstancemetadata)
+by writing a configuration to the `user-data` key.
```

Comment:
Please add.
"You can write a configuration from the command line or from the Cloud Platform Console. Both methods are described in the following sections."
Or words to that effect.